extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -308,6 +308,11 @@ public function getWikiVariables() {
 			$wikiVariables['facebookAppId'] = $egFacebookAppId;
 		}
 
+		$wikiImages = (new WikiService())->getWikiImages( [$this->wg->CityId], 500 );
+		if ( is_array( $wikiImages ) && is_string( $wikiImages[$this->wg->CityId] ) ) {
```

Comment:
I think PHP raises a Notice if you try to access `$wikiImages` as an array and it's not an array.